practice-02/README.md

```diff
@@ -0,0 +1,58 @@
+# JavaScript #Practice-02
+## Overview
+#### INTRODUCE
+- Creates a phone list, which lists the phones by filter and displays the basic information of each phone.
+- Use the manufacturer's filter to display manufacturer's products.
+- Use price filters to display the price segment of each product
+#### DESIGN
+- This is the design of the app : [link](https://drive.google.com/file/d/1XpJmiGw6D0fnW-4didAcc8v_aNNGkc4J/view?usp=drive_link)
+---
+## TIMELINE

```

Comment:
code style

<img width="536" alt="Feat___Setting_practice_by_TheHienDuong_·_Pull_Request__48_·_Agility-Internship_Duong-The-Hien-Javascript" src="https://github.com/Agility-Internship/Duong-The-Hien-Javascript/assets/27277174/446beba8-26c3-4a8d-929b-44673ee07b61">